demisto_sdk/commands/test_content/tests/BuildContext_test.py

```diff
@@ -13,7 +13,8 @@ def generate_test_configuration(playbook_id: str,
                                 toversion: str = '',
                                 timeout: int = None,
                                 memory_threshold: int = None,
-                                pid_threshold: int = None
```

Comment:
please add docstring to this func and for all args with default value of `None` set the type int to be `Optional[...]`